src/Illuminate/Database/Query/Grammars/MySqlGrammar.php

```diff
@@ -514,4 +514,15 @@ protected function wrapJsonBooleanSelector($value)
 
         return 'json_extract('.$field.$path.')';
     }
+
+    public function getConnectionCount(): ?int
+    {
+        $result = $this->connection->selectOne('show status where variable_name = "threads_connected"');
+
+        if (! $result) {
+            return null;
+        }
+
+        return (int) $result['Value'];
+    }
```

Comment:
please move this from `Query/Grammars` to `Schema/Grammar`, define this as follow:

```php
public function compileConnectionCount(): string
{
    return 'select variable_value as `Value` from performance_schema.session_status where variable_name = "threads_con
nected"';
}
```

Reply:
@GromNaN I actually suggestes new query here, to be able to use this with `scalar`, and please move this to `Shema/Grammar`.